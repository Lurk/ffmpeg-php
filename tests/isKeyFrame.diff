001+ Fatal error: failed to convert frame to gd image in /src/ffmpeg-php-code/ffmpeg-php/tests/isKeyFrame.php on line 9
001- ffmpeg frame->isKeyFrame(1)= 0
002- ffmpeg frame->isKeyFrame(2)= 0
003- ffmpeg frame->isKeyFrame(3)= 1
004- ffmpeg frame->isKeyFrame(4)= 0
005- ffmpeg frame->isKeyFrame(5)= 0
006- ffmpeg frame->isKeyFrame(6)= 1
007- ffmpeg frame->isKeyFrame(7)= 0
008- ffmpeg frame->isKeyFrame(8)= 0
009- ffmpeg frame->isKeyFrame(9)= 1
010- ffmpeg frame->isKeyFrame(10)= 0
011- ffmpeg frame->isKeyFrame(11)= 0
012- ffmpeg frame->isKeyFrame(12)= 1
013- ffmpeg frame->isKeyFrame(13)= 0
014- ffmpeg frame->isKeyFrame(14)= 0
015- ffmpeg frame->isKeyFrame(15)= 1
016- ffmpeg frame->isKeyFrame(16)= 0
017- ffmpeg frame->isKeyFrame(17)= 0
018- ffmpeg frame->isKeyFrame(18)= 1
019- ffmpeg frame->isKeyFrame(19)= 0
020- ffmpeg frame->isKeyFrame(20)= 0
021- ffmpeg frame->isKeyFrame(21)= 1
022- ffmpeg frame->isKeyFrame(22)= 0
023- ffmpeg frame->isKeyFrame(23)= 0
024- ffmpeg frame->isKeyFrame(24)= 1
025- ffmpeg frame->isKeyFrame(25)= 0
026- ffmpeg frame->isKeyFrame(26)= 0
027- ffmpeg frame->isKeyFrame(27)= 1
028- ffmpeg frame->isKeyFrame(28)= 0
029- ffmpeg frame->isKeyFrame(29)= 0
030- ffmpeg frame->isKeyFrame(30)= 1
031- ffmpeg frame->isKeyFrame(31)= 0
032- ffmpeg frame->isKeyFrame(32)= 0
033- ffmpeg frame->isKeyFrame(33)= 1
034- ffmpeg frame->isKeyFrame(34)= 0
035- ffmpeg frame->isKeyFrame(35)= 0
036- ffmpeg frame->isKeyFrame(36)= 1
037- ffmpeg frame->isKeyFrame(37)= 0
038- ffmpeg frame->isKeyFrame(38)= 0
039- ffmpeg frame->isKeyFrame(39)= 1
040- ffmpeg frame->isKeyFrame(40)= 0
041- ffmpeg frame->isKeyFrame(41)= 0
042- ffmpeg frame->isKeyFrame(42)= 1
043- ffmpeg frame->isKeyFrame(43)= 0
044- ffmpeg frame->isKeyFrame(44)= 0
045- ffmpeg frame->isKeyFrame(45)= 1
046- ffmpeg frame->isKeyFrame(46)= 0
047- ffmpeg frame->isKeyFrame(47)= 0
048- ffmpeg frame->isKeyFrame(48)= 1
049- ffmpeg frame->isKeyFrame(49)= 0
050- ffmpeg frame->isKeyFrame(50)= 0
051- ffmpeg frame->isKeyFrame(51)= 1
052- ffmpeg frame->isKeyFrame(52)= 0
053- ffmpeg frame->isKeyFrame(53)= 0
054- ffmpeg frame->isKeyFrame(54)= 1
055- ffmpeg frame->isKeyFrame(55)= 0
056- ffmpeg frame->isKeyFrame(56)= 0
057- ffmpeg frame->isKeyFrame(57)= 1
058- ffmpeg frame->isKeyFrame(58)= 0
059- ffmpeg frame->isKeyFrame(59)= 0
060- ffmpeg frame->isKeyFrame(60)= 1
061- ffmpeg frame->isKeyFrame(61)= 0
062- ffmpeg frame->isKeyFrame(62)= 0
063- ffmpeg frame->isKeyFrame(63)= 1
064- ffmpeg frame->isKeyFrame(64)= 0
065- ffmpeg frame->isKeyFrame(65)= 0
066- ffmpeg frame->isKeyFrame(66)= 1
067- ffmpeg frame->isKeyFrame(67)= 0
068- ffmpeg frame->isKeyFrame(68)= 0
069- ffmpeg frame->isKeyFrame(69)= 1
070- ffmpeg frame->isKeyFrame(70)= 0
071- ffmpeg frame->isKeyFrame(71)= 0
072- ffmpeg frame->isKeyFrame(72)= 1
073- ffmpeg frame->isKeyFrame(73)= 0
074- ffmpeg frame->isKeyFrame(74)= 0
075- ffmpeg frame->isKeyFrame(75)= 1
076- ffmpeg frame->isKeyFrame(76)= 0
077- ffmpeg frame->isKeyFrame(77)= 0
078- ffmpeg frame->isKeyFrame(78)= 1
079- ffmpeg frame->isKeyFrame(79)= 0
080- ffmpeg frame->isKeyFrame(80)= 0
081- ffmpeg frame->isKeyFrame(81)= 1
082- ffmpeg frame->isKeyFrame(82)= 0
083- ffmpeg frame->isKeyFrame(83)= 0
084- ffmpeg frame->isKeyFrame(84)= 1
085- ffmpeg frame->isKeyFrame(85)= 0
086- ffmpeg frame->isKeyFrame(86)= 0
087- ffmpeg frame->isKeyFrame(87)= 1
088- ffmpeg frame->isKeyFrame(88)= 0
089- ffmpeg frame->isKeyFrame(89)= 0
090- ffmpeg frame->isKeyFrame(90)= 1
091- ffmpeg frame->isKeyFrame(91)= 0
092- ffmpeg frame->isKeyFrame(92)= 0
093- ffmpeg frame->isKeyFrame(93)= 1
094- ffmpeg frame->isKeyFrame(94)= 0
095- ffmpeg frame->isKeyFrame(95)= 0
096- ffmpeg frame->isKeyFrame(96)= 1
097- ffmpeg frame->isKeyFrame(97)= 0
098- ffmpeg frame->isKeyFrame(98)= 0
099- ffmpeg frame->isKeyFrame(99)= 1
100- ffmpeg frame->isKeyFrame(100)= 0
101- ffmpeg frame->isKeyFrame(101)= 0
102- ffmpeg frame->isKeyFrame(102)= 1
103- ffmpeg frame->isKeyFrame(103)= 0
104- ffmpeg frame->isKeyFrame(104)= 0
105- ffmpeg frame->isKeyFrame(105)= 1
106- ffmpeg frame->isKeyFrame(106)= 0
107- ffmpeg frame->isKeyFrame(107)= 0
108- ffmpeg frame->isKeyFrame(108)= 1
109- ffmpeg frame->isKeyFrame(109)= 0
110- ffmpeg frame->isKeyFrame(110)= 0
111- ffmpeg frame->isKeyFrame(111)= 1
112- ffmpeg frame->isKeyFrame(112)= 0
113- ffmpeg frame->isKeyFrame(113)= 0
114- ffmpeg frame->isKeyFrame(114)= 1
115- ffmpeg frame->isKeyFrame(115)= 0
116- ffmpeg frame->isKeyFrame(116)= 0
117- ffmpeg frame->isKeyFrame(117)= 1
118- ffmpeg frame->isKeyFrame(118)= 0
119- ffmpeg frame->isKeyFrame(119)= 0
120- ffmpeg frame->isKeyFrame(120)= 1
121- ffmpeg frame->isKeyFrame(121)= 0
122- ffmpeg frame->isKeyFrame(122)= 0
123- ffmpeg frame->isKeyFrame(123)= 1
124- ffmpeg frame->isKeyFrame(124)= 0
125- ffmpeg frame->isKeyFrame(125)= 0
126- ffmpeg frame->isKeyFrame(126)= 1
127- ffmpeg frame->isKeyFrame(127)= 0
128- ffmpeg frame->isKeyFrame(128)= 0
129- ffmpeg frame->isKeyFrame(129)= 1
130- ffmpeg frame->isKeyFrame(130)= 0
131- ffmpeg frame->isKeyFrame(131)= 0
132- ffmpeg frame->isKeyFrame(132)= 1
133- ffmpeg frame->isKeyFrame(133)= 0
134- ffmpeg frame->isKeyFrame(134)= 0
135- ffmpeg frame->isKeyFrame(135)= 1
136- ffmpeg frame->isKeyFrame(136)= 0
137- ffmpeg frame->isKeyFrame(137)= 0
138- ffmpeg frame->isKeyFrame(138)= 1
139- ffmpeg frame->isKeyFrame(139)= 0
140- ffmpeg frame->isKeyFrame(140)= 0
141- ffmpeg frame->isKeyFrame(141)= 1
142- ffmpeg frame->isKeyFrame(142)= 0
143- ffmpeg frame->isKeyFrame(143)= 0
144- ffmpeg frame->isKeyFrame(144)= 1
145- ffmpeg frame->isKeyFrame(145)= 0
146- ffmpeg frame->isKeyFrame(146)= 0
147- ffmpeg frame->isKeyFrame(147)= 1
148- ffmpeg frame->isKeyFrame(148)= 0
149- ffmpeg frame->isKeyFrame(149)= 0
150- ffmpeg frame->isKeyFrame(150)= 1
151- ffmpeg frame->isKeyFrame(151)= 0
152- ffmpeg frame->isKeyFrame(152)= 0
153- ffmpeg frame->isKeyFrame(153)= 1
154- ffmpeg frame->isKeyFrame(154)= 0
155- ffmpeg frame->isKeyFrame(155)= 0
156- ffmpeg frame->isKeyFrame(156)= 1
157- ffmpeg frame->isKeyFrame(157)= 0
158- ffmpeg frame->isKeyFrame(158)= 0
159- ffmpeg frame->isKeyFrame(159)= 1
160- ffmpeg frame->isKeyFrame(160)= 0
161- ffmpeg frame->isKeyFrame(161)= 0
162- ffmpeg frame->isKeyFrame(162)= 1
163- ffmpeg frame->isKeyFrame(163)= 0
164- ffmpeg frame->isKeyFrame(164)= 0
165- ffmpeg frame->isKeyFrame(165)= 1
166- ffmpeg frame->isKeyFrame(166)= 0
167- ffmpeg frame->isKeyFrame(167)= 0
168- ffmpeg frame->isKeyFrame(168)= 1
169- ffmpeg frame->isKeyFrame(169)= 0
170- ffmpeg frame->isKeyFrame(170)= 0
171- ffmpeg frame->isKeyFrame(171)= 1
172- ffmpeg frame->isKeyFrame(172)= 0
173- ffmpeg frame->isKeyFrame(173)= 0
174- ffmpeg frame->isKeyFrame(174)= 1
175- ffmpeg frame->isKeyFrame(175)= 0
176- ffmpeg frame->isKeyFrame(176)= 0
177- ffmpeg frame->isKeyFrame(177)= 1
178- ffmpeg frame->isKeyFrame(178)= 0
179- ffmpeg frame->isKeyFrame(179)= 0
180- ffmpeg frame->isKeyFrame(180)= 1
181- ffmpeg frame->isKeyFrame(181)= 0
182- ffmpeg frame->isKeyFrame(182)= 0
183- ffmpeg frame->isKeyFrame(183)= 1
184- ffmpeg frame->isKeyFrame(184)= 0
185- ffmpeg frame->isKeyFrame(185)= 0
186- ffmpeg frame->isKeyFrame(186)= 1
187- ffmpeg frame->isKeyFrame(187)= 0
188- ffmpeg frame->isKeyFrame(188)= 0
189- ffmpeg frame->isKeyFrame(189)= 1
190- ffmpeg frame->isKeyFrame(190)= 0
191- ffmpeg frame->isKeyFrame(191)= 0
192- ffmpeg frame->isKeyFrame(192)= 1
193- ffmpeg frame->isKeyFrame(193)= 0
194- ffmpeg frame->isKeyFrame(194)= 0
195- ffmpeg frame->isKeyFrame(195)= 1
196- ffmpeg frame->isKeyFrame(196)= 0
197- ffmpeg frame->isKeyFrame(197)= 0
198- ffmpeg frame->isKeyFrame(198)= 1
199- ffmpeg frame->isKeyFrame(199)= 0
200- ffmpeg frame->isKeyFrame(200)= 0
201- ffmpeg frame->isKeyFrame(201)= 1
202- ffmpeg frame->isKeyFrame(202)= 0
203- ffmpeg frame->isKeyFrame(203)= 0
204- ffmpeg frame->isKeyFrame(204)= 1
205- ffmpeg frame->isKeyFrame(205)= 0
206- ffmpeg frame->isKeyFrame(206)= 0
207- ffmpeg frame->isKeyFrame(207)= 1
208- ffmpeg frame->isKeyFrame(208)= 0
209- ffmpeg frame->isKeyFrame(209)= 0
210- ffmpeg frame->isKeyFrame(210)= 1
211- ffmpeg frame->isKeyFrame(211)= 0
212- ffmpeg frame->isKeyFrame(212)= 0
213- ffmpeg frame->isKeyFrame(213)= 1
214- ffmpeg frame->isKeyFrame(214)= 0
215- ffmpeg frame->isKeyFrame(215)= 0
216- ffmpeg frame->isKeyFrame(216)= 1
217- ffmpeg frame->isKeyFrame(217)= 0
218- ffmpeg frame->isKeyFrame(218)= 0
219- ffmpeg frame->isKeyFrame(219)= 1
220- ffmpeg frame->isKeyFrame(220)= 0
221- ffmpeg frame->isKeyFrame(221)= 0
222- ffmpeg frame->isKeyFrame(222)= 1
223- ffmpeg frame->isKeyFrame(223)= 0
224- ffmpeg frame->isKeyFrame(224)= 0
225- ffmpeg frame->isKeyFrame(225)= 1
226- ffmpeg frame->isKeyFrame(226)= 0
227- ffmpeg frame->isKeyFrame(227)= 0
228- ffmpeg frame->isKeyFrame(228)= 1
229- ffmpeg frame->isKeyFrame(229)= 0
230- ffmpeg frame->isKeyFrame(230)= 0
231- ffmpeg frame->isKeyFrame(231)= 1
232- ffmpeg frame->isKeyFrame(232)= 0
233- ffmpeg frame->isKeyFrame(233)= 0
234- ffmpeg frame->isKeyFrame(234)= 1
235- ffmpeg frame->isKeyFrame(235)= 0
236- ffmpeg frame->isKeyFrame(236)= 0
237- ffmpeg frame->isKeyFrame(237)= 1
238- ffmpeg frame->isKeyFrame(238)= 0
239- ffmpeg frame->isKeyFrame(239)= 0
240- ffmpeg frame->isKeyFrame(240)= 1
241- ffmpeg frame->isKeyFrame(241)= 0
242- ffmpeg frame->isKeyFrame(242)= 0