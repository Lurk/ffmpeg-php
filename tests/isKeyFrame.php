<?php
$mov = new ffmpeg_movie(dirname(__FILE__) . '/test_media/robot.avi');
$img = dirname(__FILE__) . '/test-';
$i = 0;

while (($frame = $mov->getFrame()) != false) {
    $i = $mov->getFrameNumber();
    $filename = $img . $i . '.png';
    $gd_image = $frame->toGDImage();
    imagepng($gd_image, $filename);
    printf("ffmpeg frame->isKeyFrame($i)= %d\n", $frame->isKeyFrame());
    imagedestroy($gd_image);
    unlink($filename);
}
?>
