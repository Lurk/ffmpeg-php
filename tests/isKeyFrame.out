Fatal error: failed to convert frame to gd image in /src/ffmpeg-php-code/ffmpeg-php/tests/isKeyFrame.php on line 9